### ISSUE TITLE
Indentation errors stop better-exceptions from returning correct errors.

### ISSUE THREAD

**Issue by sentientmachine** [Qix-/better-exceptions#49]
I'm using gentoo Linux x86_64 on a PC desktop.

Using `Python 2.7.10`

Using better-exceptions version `0.1.8`

I installed as non root user: `pip install -U better_exceptions`

Put this in a file: 

    import better_exceptions
    if (True):
        a = 5
           print("foobar")      #intentional faulty indentation here.
        b = 7

It works as I expect, the better_exception defaults to normal python interpreter telling me line 4 has bad indentation.  Great.

But when I put the import in my `/home/el/.local/lib64/python2.7/site-packages/usercustomize.py` then better_exceptions charges ahead, and fails, returning: 

    Traceback (most recent call last):
    File "a.py", line 4

So better_exceptions isn't wrong, however, it failed to defer to regular python error messaging letting me know about bad indentation. 

Perhaps the problem is the way I'm importing using the usercustomize.py script.  Obviously your code is working, just not when the import is specified in a pre-processing file.

**Comment by Delgan** [comment]
Hey @sentientmachine, thank you for reporting this. 👍 

Just to let you know: this a known bug due to a small mistake while formatting, see https://github.com/Qix-/better-exceptions/pull/35.

**Comment by sentientmachine** [comment]
Wow that worked.  I applied the fix as above and now the error message is:

```
$ python c.py 
Traceback (most recent call last):
  File "c.py", line 3
    print("foobar")      #intentional faulty indentation here.
    ^
IndentationError: unexpected indent
```

**Comment by Delgan** [comment]
Well, I guess it was not merged because it did not pass the travis tests. And it did not pass the travis tests because of a `\n` remaining at the end of the string, as `.strip()` is not called.
#35 is now conflicting with master, but I was thinking of fixing it for an upcoming release.

**Comment by Delgan** [comment]
Fixed with #48. :)
